resources/lib/lists.py

```diff
@@ -1,3 +1,4 @@
+# vi: set noet ff=dos :

```

Comment:
Added this modeline as opposed to correcting indentation and newlines throughout the project.